aptos-move/aptos-gas-calibration/src/solve.rs

```diff
@@ -12,6 +12,9 @@ use crate::{
 use nalgebra::DMatrix;
 use std::collections::BTreeMap;
 
+const MAX_EXECUTION_GAS: u64 = 20000000000;
```

Comment:
Consider reading the value from `TransactionGasParameters::initial()` instead of hardcoding it?